ingest-asset-reconciler/src/test/scala/uk/gov/nationalarchives/ingestassetreconciler/testUtils/ExternalServicesTestUtils.scala

```diff
@@ -302,18 +318,7 @@ class ExternalServicesTestUtils(dynamoServer: WireMockServer) extends AnyFlatSpe
 
   private val defaultIoWithIdentifier =
     IO.pure(
-      Seq(
-        Entity(
-          Some(InformationObject),
-          UUID.fromString("354f47cf-3ca2-4a4e-8181-81b714334f00"),
-          None,
-          None,
-          false,
-          Some(InformationObject.entityPath),
-          None,
-          Some(UUID.fromString("a9e1cae8-ea06-4157-8dd4-82d0525b031c"))
-        )
-      )
+      Seq(defaultEntity)
```

Comment:
Since you've saved the entity to a variable, you could probably move this to one line `IO.pure(Seq(defaultEntity))`